server/models/Event.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+// mongoose.connect(
+//   "mongodb+srv://dmruds123:Radius335841482@pinneddb.ulvjrsw.mongodb.net/?retryWrites=true&w=majority"
+// );
+
+const eventSchema = new mongoose.Schema({
+  eventTitle: {
+    type: String,
+    required: true,
+  },
+  eventDescription: {
+    type: String,
+    required: true,
+  },
+  eventContact: String,
+  eventTags: [String],
+  eventDate: Date,
+  eventTime: String,
+  eventLocation: String,
+  eventPreview: String,
+});
```

Comment:
add timestamps to the schema, so we can sort them when organizing on front end. Add a second argument to the schema. `mongoose.Schema({...}, {timestamps: true})`